kotsadm/operator/Dockerfile.skaffold

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16 as deps
+FROM kotsadm-operator:cache AS builder
```

Comment:
This should use multistage to reduce image size.  We don't need all dependencies in the running image; just the operator binary.  Similar to this https://github.com/replicatedhq/kots/blob/master/hack/dev/Dockerfile.skaffold#L86